### DISCUSSION TITLE
Should SBE participate in hacktoberfest?

### DISCUSSION THREAD

**Discussion by kohasummons** [kohasummons/soroban-by-example#47] [General]
The title says it all. Good exposure. Possibly spammy PRs.

**Comment by koolamusic** [comment]
Yeah of course we'll get spammy PRs. 

I don't see why we can't give it a shot though. It's the ideal Hacktoberfest project. 

Lean setup
Documentation focused
Simple & small objective. 

So many Good first issue potential
